Add import of default clone function (#15)
davidmarkclements/rfdc: readme.md

### `require('rfdc')(opts = { proto: false, circles: false }) => clone(obj) => obj2`

> #### `default` option
> 
> If you just need the default implementation of the clone function without any options,
> you can directly import it by calling:

Reviewer: It's not really a option, I would therefore move the section below the `circles` option and change the description as follows:

```suggestion
### `default` import

It is also possible to directly import the clone function with all options set
to their default:
```
